index.html

```diff
@@ -20,11 +21,9 @@ <h2><a class="white pen" href="#">Kenny Elias</a></h2>
                 
                 <li class="list list2 white"><a class="white" href="#works">Portfolio</a></li>
                 
-                <li class="list list4 white"><a class="white" href="#contact">Contact Us</a></li>
+                <li class="list list4 white"><a class="white" href="#contact">Contact</a></li>
                 
                 <li class="list list3 white"><a class="white" href="#about">About</a></li>
-                
-                <li class="lis white"><a href=""><img src="enve.svg" alt=""></a></li>
             </ul>
```

Comment:
- [ ] Great job here 💯 However, According to the requirements you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). so Kindly add some margin-top or padding-top between nav items 👌 

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/188540401-26650cd3-a8a5-4ddf-8c8e-6ef3d84e0396.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/188540484-edfebe49-7639-49d1-8523-64902d30c483.png" width="500" /> |

Reply:
Fixed now

Reply:
Hey @kennyegun24, as pointed out by the previous reviewer, this problem still persists. Have a look at the current state of your application. ⬇️. You can fix this issue by giving `padding` or a `margin` to the individual elements inside the class `.menu .in-active`. In your case some thing like this: `@media (max-width: 765px)
.list {
  line-height: 44px;
  padding-top: 40px;
}`
![image](https://user-images.githubusercontent.com/33268927/188548908-dc4e6f06-2cfb-4452-8280-78b4392d15c3.png)

Reply:
I fixed that already by giving the list items a `padding-top of 10px` which gave the items spaces between them. Padding of 40px is way too much and will give them too much space. So I just increased it to 12px, just to give them a better resemblance